src/main/java/com/tomcat/dao/StudentDao.java

```diff
@@ -0,0 +1,17 @@
+package com.tomcat.dao;
+
+import com.tomcat.entity.Student;
+
+import java.util.List;
+
+public interface StudentDao {
+    void insertStudent(Student student);
```

Comment:
You have duplicate word student. You have it already in the name of dao.
insertStudent ->insert (or create)
updateStudent -> update
deleteStudent->deleteById
getByIdStudent-> getById (or findById)
getAllStudents ->getAll (or findAll)